### ISSUE TITLE
Revamping audio/UI related events

### ISSUE THREAD

**Issue by oncletom** [bbc/peaks.js#18]
# tl;dr

The UI should drive events only, subscribed and handled by views.
# Interface to Internals Events
- `waveform_seek`
- `waveform_zoom_level_changed`
- `waveform_size_changed` (originally `window_resized`)

Events should contain:
- time
- currentFrame
- origin
# Internals to Interface Events
- `player_time_update`
- `player_play`
- `player_pause`
- `zoomview_resized`
- `player_ready`
- `waveform_ready`
- `peaks_ready` (will be a separate issue later on)
# Goals
- UI will collect interactions and spread them across the views
- avoiding side-effects caused by cascaded events
- being able to drive the UI through events only
- driving the interface just by acting on the media element

**Comment by chrisn** [comment]
Closing this as resolved in the [v0.9.0](https://github.com/bbc/peaks.js/releases/tag/v0.9.0) release. This decouples the views from the points and segments APIs and uses events to drive UI updates.
